src/main/java/core/basesyntax/helpers/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.helpers;
+
+import core.basesyntax.enums.Color;
+import core.basesyntax.interfaces.RandomNumber;
+import java.util.Random;
+
+public class ColorSupplier implements RandomNumber {
```

Comment:
ColorSupplier should not have getRandomNumber method
Add Random as class field and use it
```suggestion
public class ColorSupplier {
```

Reply:
++